### PR TITLE
Make EXIF data displayed configurable

### LINKED ISSUE #11: Descriptions on photos (?)
Hi, 

I'm thinking of adding the ability to add some text descriptions to each photo.  Do you have any thoughts on this?

I was thinking of, but not really checked what is possible with Jekyll and the lightbox you are using.
 
- Another file named the same as the full/image file that contains the html to display
- If this new file exists, then display first line or 2 of the text on top of the image (similar to how the exif data is displayed)
- When opened in the lightbox, display all of the text details under the image.

Just looking for feedback on this, before I start to tinker around with it.

Sam

### LINKED ISSUE #23: Modifying EXIF displayed?
Thanks so much for making this available! 

I'm wondering if you could edit the ReadMe to provide some more information about how to modify the EXIF. It looks, from `assets/js/exif.js` that there are a number of options?

### PR DESCRIPTION
**Pull request by Niduroki**
Closes #11 via EXIF UserComment and Closes #23

I only tested this with my own site, but it works well enough over there.
I didn't really test anything but the default so it might break if one were to put loads of stuff into the exif-data footer and/or has an exif user-comment that is pretty long.
People will just have to figure it out I guess.

### CODE REVIEW

**Comment by rampatra** [comment]
I guess it may be beneficial to add some screenshots if you have them handy? Or, if you have implemented this on your website and you're okay to share your link then it would be great too.

**Review by rampatra — COMMENTED** [review]
Looks good to me. If you reopen this pull request, I can merge it.

File: _config.yml [hunk 1]
```diff
@@ -23,3 +23,7 @@ social_urls:
   dribbble: "https://dribbble.com/rampatra"
   linkedin: "https://in.linkedin.com/in/ram-patra"
   github: "https://www.github.com/rampatra"
+
+# Config which exif data to display
+# Tag is the actual exif tag, icon is a fontawesome icon. Use JSON notation without line breaks
+exif_display: '[{"tag": "Model", "icon": "camera-retro"}, {"tag": "FNumber", "icon": "dot-circle-o"}, {"tag": "ExposureTime", "icon": "clock-o"}, {"tag": "ISOSpeedRatings", "icon": "info-circle"}]'
```
Comment (rampatra) [inline _config.yml:29]: Maybe a better name would be `exif_tags`.

File: assets/js/main.js [hunk 2]
```diff
@@ -307,46 +307,18 @@
             });
 
         function getExifDataMarkup(img) {
-            var exif = fetchExifData(img);
+            var exif_display = $('#main').data('exif-display');
```
Comment (rampatra) [inline assets/js/main.js:310]: Same here.

**Comment by Niduroki** [comment]
If you (after 3,5 years of no comment) still care about this PR feel free to copy the provided code and merge it yourself.

This PR can be considered as MIT-licensed, thus do what you want with the code.
There is no way of reopening this PR, since I have deleted the original repo.

**Comment by rampatra** [comment]
Changes added to master!

**Pull request closed without merging**